lib/utils/index.js

```diff
@@ -2536,6 +2537,31 @@ function getVueComponentDefinitionType(node) {
       if (callee.name === 'createApp') {
         // for Vue.js 3.x
         // createApp({})
+
+        const variable = findVariable(context.getScope(), callee)
```

Comment:
Sorry for the late reply. I am busy with office work.

I think it can be written more simply using `tracker.iterateEsmReferences`.

https://eslint-community.github.io/eslint-utils/api/scope-utils.html#tracker-iterateesmreferences

e.g.

```js
          const tracker = new ReferenceTracker(context.getSourceCode().scopeManager.globalScope)
          const traceMap = utils.createCompositionApiTraceMap({
            [ReferenceTracker.ESM]: true,
            createApp: {
              [ReferenceTracker.CALL]: true
            }
          })
          if ([...tracker.iterateEsmReferences(traceMap)].every(({ node }) => node !== callee)) {
            return null;
          }
```

Reply:
@candy-Tong Would you like to give this a try?

Reply:
@candy-Tong Would you like to give this a try?

Reply:
@candy-Tong Would you like to give this a try?